www/nginx/src/opnsense/mvc/app/models/OPNsense/Nginx/Nginx.xml

```diff
@@ -514,6 +542,23 @@
         <Required>N</Required>
         <default>443</default>
       </listen_https_port>
+      <proxy_protocol type="BooleanField">
+        <default>0</default>
+        <Required>Y</Required>
+      </proxy_protocol>
+      <trusted_proxies type="CSVListField">
+        <Required>N</Required>
+        <mask>/^((?:\d+\.){3,3}\d+|[a-f0-9\:]+)(?:\/\d+)?(,?(?:(?:(\d+\.){3,3}\d+|[a-f0-9\:]+)(?:\/\d+)?))*$/i</mask>
+        <multiple>Y</multiple>
+      </trusted_proxies>
+      <real_ip_source type="OptionField">
+        <OptionValues>
+          <X-Real-IP>X-Real-IP (default)</X-Real-IP>
```

Comment:
Shouldn't there be a <default>?

Reply:
Your comment is incomplete…

Reply:
fixed the comment ;)

Reply:
No, the default is configuring nothing. If nothing is configured, nginx will fall back to its default, which currently is X-Real-IP and may change in the future. If you choose the option it will be set explicitly in the configuration and will probably be future safe against changes (regarding unexpected changes).